src/config/etc/sssd.api.d/sssd-ipa.conf

```diff
@@ -1,10 +1,13 @@
+# Format:
+# option = type, subtype, mandatory[, default]
 [provider/ipa]
 ipa_domain = str, None, false
 ipa_server = str, None, false
 ipa_backup_server = str, None, false
 ipa_hostname = str, None, false
 ipa_deskprofile_search_base = str, None, false
 ipa_subid_ranges_search_base = str, None, false
+ipa_access_order = str, None, None,
```

Comment:
Hi,

this has to be `ipa_access_order = str, None, false` since `mandatory` expects a boolean.

bye,
Sumit

Reply:
Oh my gosh.
Fixed.